test/time-utils.spec.js

```diff
@@ -54,3 +52,55 @@ test('there are closed notify before stale date', () => {
 
   expect(stale).toBe(true)
 })
+
+test('convert notify after to date', () => {
+  const now = Date.now()
+  const spy = jest.spyOn(Date, 'now').mockImplementation(() => now)
+
+  const defaultStaleDate = () => notifyAfterToMs()
+  expect(defaultStaleDate).toThrow()
+
+  const oneHourAgo = notifyAfterToMs('1 hour')
+  expect(oneHourAgo).toEqual(now - 60 * 60 * 1000)
+
+  spy.mockRestore()
```

Comment:
let's use a more robust approach at restoring mocks please, so that you don't have to remember to do it after each test, plus, if the test fails the mock is not restored. there are several ways that jest can do this without having to restore mocks manually after each test

Reply:
I've decided to use the [clearMocks](https://jestjs.io/docs/configuration#clearmocks-boolean) to reset every mock after each test